crates/holochain/src/conductor/conductor/tests/space_config_override.rs

```diff
@@ -0,0 +1,117 @@
+use holochain_p2p::Tx5TransportModConfig;
+use holochain_types::app::{AppManifest, AppManifestV0};
+use kitsune2_core::factories::CoreBootstrapModConfig;
+use kitsune2_test_utils::bootstrap::TestBootstrapSrv;
+
+use crate::{conductor::Conductor, sweettest::SweetConductor};
+
+#[tokio::test(flavor = "multi_thread")]
+async fn test_should_override_space_config() {
+    holochain_trace::test_run();
+    let (dna, _, _) = crate::conductor::conductor::tests::mk_dna(
+        crate::test_utils::inline_zomes::simple_crud_zome(),
+    )
+    .await;
+    let mut conductor = SweetConductor::from_standard_config().await;
+
+    let role_name = "role".to_string();
+
+    // create a test bootstrap server
+    let boostrap_srv = TestBootstrapSrv::new(false).await;
+    let bootstrap_server_url = boostrap_srv.addr().to_string();
+
+    let app_id = "app_id".to_string();
+    let manifest = AppManifest::V0(AppManifestV0 {
+        allow_deferred_memproofs: false,
+        description: None,
+        name: "dummy".to_string(),
+        roles: vec![],
+        bootstrap_url: Some(bootstrap_server_url),
+        signal_url: None,
+    });
+
+    conductor
+        .install_app_with_manifest(&app_id, None, [&(role_name.clone(), dna)], None, manifest)
+        .await
+        .expect("failed to install app");
+
+    conductor
+        .enable_app(app_id.clone())
+        .await
+        .expect("failed to enable app");
```

Comment:
Can you assert the bootstrap URL here with something like `conductor.get_rendezvous_config().unwrap().bootstrap_addr()`?

Reply:
Here I can't get  the rendezvous config. It returns None.

```rust
let conductor_bootstrap_addr = conductor
        .get_rendezvous_config()
        .expect("failed to get rendezvous config")
        .bootstrap_addr()
        .to_string();
```